anecdote/assets/logs/anecdote.yaml

```diff
@@ -0,0 +1,55 @@
+id: anecdote
+metric_id: anecdote
+backend_only: true
+pipeline:
+  type: pipeline
+  name: Anecdote Log Pipeline
+  enabled: true
+  filter:
+    query: "source:anecdote"
+  processors:
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
+    - type: date-remapper
+      name: Define `content.timestamp` as the official date of the log
+      enabled: true
+      sources:
+        - content.timestamp
+    - type: attribute-remapper
+      name: Map `id` to `log_id`
+      enabled: true
+      sources:
+        - id
+      sourceType: attribute
+      target: log_id
+      targetType: attribute
+      preserveSource: true
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `content.attributes.date` to `log_date`
+      enabled: true
+      sources:
+        - content.attributes.date
+      sourceType: attribute
+      target: log_date
+      targetType: attribute
+      preserveSource: true
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `content.tags` to `tags`
+      enabled: true
+      sources:
+        - content.tags
+      sourceType: attribute
+      target: tags
+      targetType: attribute
+      preserveSource: true
+      overrideOnConflict: false
+    - type: status-remapper
+      name: Define `content.attributes.is_feedback_bug` as the official status of the log
```

Comment:
`is_feedback_bug` is a boolean here. It doesn't really make sense to use this for remapping a status. I am not sure in your case you need a status?

You can see our doc about status_remapper here: https://docs.datadoghq.com/service_management/events/pipelines_and_processors/status_remapper/

Reply:
Hey, I updated log pipeline. Could you check it please?

Reply:
There is still 2 time the preserveSource to true, please set it to false. You will also need to update the test file